src/main/external-resources/renderers/Mirascreen.conf

```diff
@@ -0,0 +1,29 @@
+#----------------------------------------------------------------------------
+# Auto-generated profile for Mirascreen BFCFA04F
+# Based on DefaultRenderer.conf
+# See DefaultRenderer.conf for a description of all possible configuration options.
+#
+
+RendererName = Mirascreen
+RendererIcon = Mirascreen.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# {friendlyName=Mirascreen BFCFA04F, manufacturer= , modelName=Mirascreen BFCFA04F, modelNumber= , modelDescription=Actions-Micro DLNA DMR Using Auto-Generated UPnP Stack, manufacturerURL=http://actions-micro.com, modelURL=http://actions-micro.com}
+# ============================================================================
+
+UserAgentSearch = 
+UpnpDetailsSearch = Mirascreen BFCFA04F
+
+LoadingPriority = 3
+MediaInfo = true
+SeekByTime = true
+TranscodeVideo = MPEGTS-MPEG2-AC3
+TranscodeAudio = MP3
+DefaultVBVBufSize = true
+MaxVideoBitrateMbps = 20
+HalveBitrate = true
```

Comment:
We generally discourage the use of HalveBitrate now so I would remove that

Reply:
okay, but remove only line with HalveBitrate